src/openqaoa-pyquil/openqaoa_pyquil/backends/qaoa_pyquil_qpu.py

```diff
@@ -127,6 +127,13 @@ def __init__(
                 if initial_qubit_mapping is not None
                 else list(range(self.n_qubits))
             )
+        else:
+            if isinstance(initial_qubit_mapping, list):
+                warnings.warn(
+                    "Ignoring the initial_qubit_mapping since the routing algorithm chose one"
+                )
+            else:
```

Comment:
The `else pass` isnt needed here since the `else` statement doesnt do anything.

Reply:
fair point! Maybe its some remnant of some previous code. I'll remove it!